src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamsBaseTest.java

```diff
@@ -277,4 +287,59 @@ void testTableStream(MtAmazonDynamoDbBase mtDynamoDb) {
         }
     }
 
+    /**
+     * Verifies that reading virtual tables with streams disabled doesn't return records.
+     */
+    @ParameterizedTest
+    @ArgumentsSource(Args.class)
+    void testDisabledStreams(MtAmazonDynamoDbBase mtDynamoDb) {
+        final MtAmazonDynamoDbStreams mtDynamoDbStreams = MtAmazonDynamoDbStreams.createFromDynamo(mtDynamoDb,
+            AmazonDynamoDbLocal.getAmazonDynamoDbStreamsLocal());
+        try {
+            String tenant = TENANTS[0];
+
+            // create table with streams enabled
+            String tableWithStreamsEnabled = TENANT_TABLE_NAME + "_streams_enabled";
+            CreateTableRequest createTableRequestStreamsEnabled =
+                newCreateTableRequest(tableWithStreamsEnabled)
+                    .withStreamSpecification(new StreamSpecification()
+                        .withStreamEnabled(true)
+                        .withStreamViewType(NEW_AND_OLD_IMAGES));
+            MT_CONTEXT.withContext(tenant, () -> mtDynamoDb.createTable(createTableRequestStreamsEnabled));
+
+            // create table with streams disabled
+            String tableWithStreamsDisabled = TENANT_TABLE_NAME + "_streams_disabled";
+            CreateTableRequest createTableRequestStreamsDisabled =
+                newCreateTableRequest(tableWithStreamsDisabled)
+                    .withStreamSpecification(new StreamSpecification()
+                        .withStreamEnabled(false)
+                    );
+            MT_CONTEXT.withContext(tenant, () -> mtDynamoDb.createTable(createTableRequestStreamsDisabled));
+
+            // put an item in each table
+            final MtRecord streamsEnabledRecord = putTestItem(mtDynamoDb, tableWithStreamsEnabled, tenant, 1);
+            putTestItem(mtDynamoDb, tableWithStreamsDisabled, tenant, 2);
+
```

Comment:
Would be good to add a variant that queries the stream without context.

Reply:
yes, will do